fili-core/src/main/resources/moduleConfig.properties

```diff
@@ -53,6 +53,10 @@ bard__memcached_expiration_seconds = 3600
 # Data Cache
 bard__druid_cache_enabled = true
 
+# Maximum Druid response size to cache, in bytes. Defaults to MAX_LONG
+# If using memcached, make sure this aligns with the setting memcached is using, overridden with it's -I parameter
+bard__druid_max_response_length_to_cache = 9223372036854775807
```

Comment:
Might want to explain where this number comes from....? If it is MAX_LONG, I suggest simply use the default provided by the code and remove the constant here and simply include a commented out variable.